### PR TITLE
Update Stripe Connect Onboarding article

### PR DESCRIPTION
**Pull request by SariSaar**

### CODE REVIEW

File: src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md [hunk 1]
```diff
@@ -194,61 +204,71 @@ after the account has been created.
...
-starting point but you should keep in mind that **they will not be
-updated** by our team.
-
 You can find the deprecated files still from v.3.7.0
 
 - [PayoutDetailsForm](https://github.com/sharetribe/ftw-daily/tree/v3.7.0/src/forms/PayoutDetailsForm)
 - [PayoutPreferencesPage](https://github.com/sharetribe/ftw-daily/tree/v3.7.0/src/containers/PayoutPreferencesPage)
 - [stripe.duck.js](https://github.com/sharetribe/ftw-daily/blob/v3.7.0/src/ducks/stripe.duck.js).
+
+## Stripe for customers: saved payment methods
```
Comment (shareoc) [inline src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md:232]: We have this article: https://www.sharetribe.com/docs/how-to/save-payment-card
on how the template handles saved payment method. So maybe this section could be client agnostic

Comment (shareoc) [reply src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md:232]: Also we could mention what relevance setupIntents have in this process: https://www.sharetribe.com/api-reference/marketplace.html#create-stripe-setup-intents

**Comment by shareoc** [comment]
@SariSaar 
I would adjust the article structure under the current **Stripe for providers: Stripe Connect Onboarding** heading. The current structure reads to me as a how-to (e.g., with steps 1-4).

Here’s a possible TOC and some ideas on what it could cover — just a suggestion to consider!

**Provider onboarding in Stripe**
- Why is onboarding required for Providers, why do we recommend using Stripe hosted onboarding

  **Stripe Connect Onboarding Flow (subheading)**
  - How does this flow work from a technical perspective (client agnostic) (e.g. we Fetch Stripe Token, create stripe account, create stripe account link). 
  - Maybe links to Connect Onboarding components like StripePayoutPage and StripeConnectAccountForm 

  **Stripe Accounts (subheading)**
  - What are stripe connect accounts and how does the StripeAccount endpoint correlate
  - What Sharetribe endpoint is used and what kind of calls do we make to Stripe 
  - Current article states things like “it’s recommended for you to pass the following information in the create-stripe-account call” but doesn’t explain why. I think we should be more explicit on what info you should pass to the endpoint and why. A lot of the parameters are listed as optional in the API ref and it’s unclear why they are optional
  - What are Stripe Persons (https://www.sharetribe.com/api-reference/marketplace.html#create-stripe-person) and when are they used

  **Handling Verification Status & Stripe Account links**
    - Why does Stripe stagger requirements and how can the client know if the Stripe account needs more info
    - Modifying at which point Stripe asks for extra requirements (https://github.com/sharetribe/web-template/blob/main/src/ducks/stripeConnectAccount.duck.js#L287 e.g. modifying the currently_due parameter)
    - Maybe keep the examples from the template but we could also provide examples of what API responses correspond with each

**Comment by SariSaar** [comment]
I now rewrote the Stripe Connect Accounts and Onboarding section to be more client agnostic and consider the questions in the outline. The only thing I left out was [Stripe Persons](https://www.sharetribe.com/api-reference/marketplace.html#create-stripe-person) - the endpoint is only used if someone creates a custom onboarding flow,  and since we don't recommend it, I thought I would leave it out. The example deprecated files for the custom onboarding have examples on creating a Stripe Person, so if someone wants to use the endpoint, they will find the instructions in the files.

File: src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+For both Sharetribe Web Template and custom implementations, your
+marketplace will first need to set up Stripe for payments and enable
+Connect Onboarding. You can review the
+[instructions in our Help Center](https://www.sharetribe.com/help/en/articles/8413086-how-to-set-up-stripe-for-payments-on-your-marketplace)
+for more details.
+
+### Stripe Connect Custom Accounts
+
+Sharetribe uses Stripe Connect Custom Accounts as the default payment
```
Comment (shareoc) [inline src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md:48]:
```suggestion
Sharetribe uses Stripe Connect with Custom Accounts as the default payment
```

File: src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+or directly through
+[Stripe API](https://docs.stripe.com/api/accounts/update).
+
+### Fetching an existing Stripe Connect Account
+
+If the user already has a Stripe account, the
+[currentUser resource](https://www.sharetribe.com/api-reference/marketplace.html#currentuser-resource-format)
+has a _stripeConnected_ boolean flag set to _true._ To fetch the
+provider's existing Stripe Connect account details, we can use the
+Marketplace API
```
Comment (shareoc) [inline src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md:73]: Suggestion: consider using either `user` or `provider` in this paragraph for consistence

File: src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+[Stripe Account fetching](https://www.sharetribe.com/api-reference/marketplace.html#fetch-stripe-account)
```
Comment (shareoc) [inline src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md:74]:
```suggestion
[Stripe Account fetch](https://www.sharetribe.com/api-reference/marketplace.html#fetch-stripe-account)
```

File: src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+[create a Stripe account](https://www.sharetribe.com/api-reference/marketplace.html#create-stripe-account)
+and pass any tokens you created in the previous steps to the endpoint.
+The only mandatory parameter to create a Stripe Connect account through
+the Marketplace API is the country of the account, and it cannot be
+modified after the account has been created.
+
+In addition to the country information, we recommend passing the -
+_requestedCapabilities_ parameter – the required capabilities for
+payments to work in Sharetribe are _card_payments_ and _transfers_, so
+you will need to define these for the account for it to work with the
```
Comment (shareoc) [inline src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md:119]: Might be a formatting mistake here, one is an hyphen and one is an em dash 

> In addition to the country information, we recommend passing the -
> _requestedCapabilities_ parameter –

File: src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md [hunk 6]
```diff
@@ -0,0 +1,287 @@
+---
+title: Providers and customers on your Stripe Platform
+slug: providers-and-customers-on-stripe-platform
+updated: 2024-10-31
```
Comment (SariSaar) [inline src/docs/concepts-payments/provider-onboarding-and-identity-verification/index.md:4]: TODO: update this date before merging

**Pull request merged**